### ISSUE TITLE
segregate prod and test everything

### ISSUE THREAD

**Issue by marcidy** [noisebridge/infrastructure#221]
In order to be able to develop/ test, the database users should be different and have different privileges on different databases.  The apps need to have different ports and run from different directories.  Basically the entire processes are the same except for some privileges, users, and directories.  Would like to be able to deploy a test env.  Would be neat if someone can use ansible to deploy themselves a test environment on our servers.

**Comment by SuperQ** [comment]
Yup, as soon as we're ready to make an actual prod database, this should be easy, the role is basically already setup for this.

**Comment by marcidy** [comment]
awesome.  Hope my changes in #219 dont break that
